Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1325,34 +1325,37 @@ extension Tensor.IndexPath {
 //===------------------------------------------------------------------------------------------===//
 
 extension Tensor {
-  /// Returns `true` if the given axis is in the range `[-rank, rank)`.
+  /// Returns `true` if `axis` is in `-rank..<rank`, indicating it refers to an axis of `self`.
   @usableFromInline
-  internal func isAxisInRange<T: BinaryInteger>(_ axis: T) -> Bool {
+  internal func isInRange<T: BinaryInteger>(axis: T) -> Bool {
     let axis = Int(axis)
     return axis >= -rank && axis < rank
   }
 
-  /// Returns `true` if the given scalar tensor is in the range `[-rank, rank)`.
+  /// Returns `true` iff all elements in `axis` are in `-rank..<rank`, indicating they refer to an
+  /// axis of `self`.
```

Comment:
See remark on the scalar version's comment.

Also, this appears to say this works on arbitrary tensors of axes, when it requires a “scalar tensor” according to the `precondition` call in the body.  If the doc comment were right, I'd expect this to be `areInRange(axes:)`, so I am guessing you need to fix the comment to reflect the precondition?